test/e2e/apimachinery/garbage_collector.go

```diff
@@ -179,7 +180,7 @@ func newGCPod(name string) *v1.Pod {
 			Containers: []v1.Container{
 				{
 					Name:  "nginx",
-					Image: "gcr.io/google_containers/nginx:1.7.9",
```

Comment:
nginx-slim is not the same as nginx. was this intentional?

Reply:
`gcr.io/google-containers/nginx:1.7.9` was created 1 and  1/2 year back and don't find proper source also for this image. The `nginx-slim` image is well maintained by ingress project and see often updates as well. Moreover, I don't find any difference using this nginx-slim image anyways.

So I vote for using `nginix-slim` image for all the tests.